### ISSUE TITLE
[Adaptive Cards: Main Body]: Ensures <li> elements are used semantically

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8155]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: 22H2(OS Build 25247.1000)
Browser Version: Version 110.0.1543.0 (Official build) dev (64-bit)
Tool: Accessibility Insight For Web

**Repro Steps:**
1. Open the above URL in the browser.
2. Run the accessibility insight for web tool and observe the issue.

**Actual Result:**
<li> elements are not used semantically.

**Expected Result:**
<li> elements should be used semantically.

**User Impact:**
It will impact the screen reader user if the lists are not used semantically.

**Element path:** 
w3-row > li:nth-child(7)

**How To Fix:**
Fix any of the following:
  List item does not have a <ul>, <ol> parent element

**MAS Reference link:**
https://microsoft.sharepoint.com/:w:/s/accessibility/EXdDUZzldv5MgwDaMHnTzcMBcT8opKGncYjU-gCJ4uqQdg?e=9nuVmt

“Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-vaibgu%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400980259288%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=TrcBaCyAPKg0WAQfdSZJP2i9pghMEDg59i2NeUUs5EQ%3D&reserved=0) “

### Screenshots

![MAS1 3 1-list element are used semantically](https://user-images.githubusercontent.com/95913748/208449660-f0dd3258-5a12-420c-ba94-58f0aeb7c40f.png)
![MAS1 3 1-list element are used semantically](https://user-images.githubusercontent.com/95913748/208449675-18eb5f59-6b9d-47e7-bc03-df25f8641fe4.png)

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
This issue is not repro at our end, hence closing this issue.

**Snippet:**

![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/86a89ba6-fcbb-4b00-8bfc-e582b0d0d20c)
